### PR TITLE
feat(gatsby-telemetry): adjust how we capture plugins for telemetry

### PR DESCRIPTION
**Pull request by jamo**
This will help us better see gatsby-image utilisation

### CODE REVIEW

File: packages/gatsby-telemetry/src/get-dependencies.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    return undefined
+  }
+  return Object.entries(deps)
+    .map(([name, version]) => {
+      return {
+        name,
+        version,
+      }
+    })
+    .filter(({ name }) => name.includes(`gatsby`))
```
Comment (wardpeet) [inline packages/gatsby-telemetry/src/get-dependencies.ts:26]: can we include all of them?

Comment (jamo) [reply packages/gatsby-telemetry/src/get-dependencies.ts]: yes, updated

File: packages/gatsby-telemetry/src/get-dependencies.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { readFileSync } from "fs"
```
Comment (wardpeet) [inline packages/gatsby-telemetry/src/get-dependencies.ts:1]: we use fs-extra for better os support
```suggestion
import { readFileSync } from "fs-extra"
```

**Review by wardpeet — APPROVED** [review]

**Comment by vladar** [comment]
@jamo By default, this will be published as stable next Thursday (November 12th) in `gatsby-telemetry@1.4.0` (and `gatsby@2.26.0`). Please let me know if you need this in the current stable line (will have to publish as `patch` in this case).

**Comment by jamo** [comment]
It would be nice if we could ship this earlier, this is to help see how the new gatsby image is getting adopted. But it's not extremely time sensitive. 
I'm already afk for today but we can chat more tomorrow

**Pull request merged**